testsuite/MDAnalysisTests/coordinates/test_coordinates.py

```diff
@@ -123,12 +123,12 @@ def test_frame_numbering(self):
 
     def test_frame(self):
         self.trajectory[0]
-        coord0 = self.universe.atoms.coordinates().copy()
+        coord0 = self.universe.atoms.positions.copy()
```

Comment:
Nevermind, we discussed this. We have to make it clear that indexing must happen on the group, not the quantity, so instead we should do:

```python

u.atoms[10].position = 0

u.atoms.positions[10] == 0    # yields true
```

The other way isn't possible (to my knowledge) due to how properties work.